clang/test/SemaCXX/cxx20-using-enum.cpp

```diff
@@ -148,13 +148,10 @@ template <int I> struct C {
   enum class D { d,
                  e,
                  f };
-  using enum D;
-
-  static constexpr int W = int(f) + I;
+  using enum D; // expected-error {{using-enum cannot name a dependent type}}
```

Comment:
It looks like we always got this wrong, did this fix just fall out of the implementation?

Reply:
Yes. I first implemented the DR, ran the tests, and this one failed. Then I realized that this test was incorrect.